docs/gettinginvolved/plugins/index.rst

```diff
@@ -12,6 +12,14 @@ The documents listed in the contents below will show you how to implement a bloc
 
 Your storage driver must be a Python 2.7 library providing a class implementing the `flocker.node.agents.blockdevice.IBlockDeviceAPI <https://github.com/ClusterHQ/flocker/blob/master/flocker/node/agents/blockdevice.py>`_ interface.
 
+.. note::
+	If you want your storage driver to support :ref:`storage-profiles`, you can also implement the `flocker.node.agents.blockdevice.IProfiledBlockDeviceAPI <https://github.com/ClusterHQ/flocker/blob/master/flocker/node/agents/blockdevice.py>`_ interface.
+	
+	Please note that you must implement ``bronze``, ``silver``, and ``gold`` profiles.
+	
+	If you do choose to implement profiles, it would be great to :ref:`hear from you <talk-to-us>` about what you think of both profiles in general, and of the interface.
```

Comment:
Suggested sentence: "If you do choose to implement profiles, please don't hesitate to contact us with your feedback, comments and suggestions about to how you're using this feature and how we might continue to improve it in future."